krux_boto/boto.py

```diff
@@ -216,3 +256,51 @@ def wrapper(*args, **kwargs):
             return wrapper
 
         return attr
+
+class Boto(BaseBoto):
+
+    # All the hard work is done in the superclass. We just need to use the
+    # resulting object to initialize a session properly.
+    def __init__(self, *args, **kwargs):
```

Comment:
Instead of taking arbitrary number of arguments via `*args` and `**kwargs`, we should write each individual ones out for the ease of use and clarity.